frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.tsx

```diff
@@ -1,12 +1,43 @@
 import * as React from 'react';
+import * as _ from 'lodash';
+import { match as RMatch } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
+import { Grid, GridItem } from '@patternfly/react-core';
+import MonitoringDashboardGraph from './MonitoringDashboardGraph';
+import { queries } from './MonitoringDashboardQueries';
```

Comment:
Nit: MonitoringDashboardQueries is not a component file and therefore the file name shouldn't begin in uppercase.